pkg/utils/stringset.go

```diff
@@ -1,44 +0,0 @@
-//
```

Comment:
I am not sure of deleting files under pkg. Theoretically it is possible that other projects are using this.

Reply:
Actually, we don't need the stringset at all. The requirement of stringset went away with https://github.com/heketi/heketi/commit/d1e62d6e57d97290310a37d738dce400e33ac4cb . This code can now simplified to be 
```
// Get all nodes in the trusted storage pool(this is gluster name for cluster)
        hosts := make([]string, 0)
	if err := db.View(func(tx *bolt.Tx) error {
		cluster, err := NewClusterEntryFromId(tx, v.Info.Cluster)
		if err != nil {
			return err
		}
		for _, nodeId := range cluster.Info.Nodes {
			node, err := NewNodeEntryFromId(tx, nodeId)
			if err != nil {
				return err
			}
                        hosts = append(hosts, node.StorageHostName())
		}
		return err
	}); err != nil {
		return err
	}
	v.Info.Mount.GlusterFS.Hosts = hosts
```